src/on-create-node.js

```diff
@@ -9,10 +9,7 @@ const onCreateNodeMarkdownRemark = async ({
   cache,
   createNodeId,
 }) => {
-  if (
-    node.frontmatter.cover &&
-    GOOGLE_IMAGE_REGEX.test(node.frontmatter.cover.image)
-  ) {
+  if (node.frontmatter.cover) {
```

Comment:
I think it will be better to improve the regex. 
Could you give me some urls that don't match the regex?
I can help you on this